lib/avalon/batch/entry.rb

```diff
@@ -47,7 +47,7 @@ def media_object
             end
             # Sometimes we want to override the bib import data for some of the
             # required fields (particularly if import data doesn't pass validations)
-            mo.assign_attributes(fields.slice(:language, :topical_subject, :genre))
+            mo.assign_attributes(fields.slice(:language, :topical_subject, :genre, :terms_of_use))
```

Comment:
UAL added `:terms_of_use` as a required field. This change allows overriding `:terms_of_use` via the batch ingest document.